bitsail-components/bitsail-component-metrics/bitsail-component-metrics-prometheus/src/main/resources/META-INF/services/com.bytedance.bitsail.base.metrics.reporter.MetricReporterBuilder

```diff
@@ -0,0 +1,82 @@
+#
+# Copyright 2022 Bytedance Ltd. and/or its affiliates.
```

Comment:
too much license header?

Reply:
I refer to the file under `bitsail-base/src/main/resources/META-INF/services`, the license header is the same, do it need to be changed?

Reply:
looks like unnecessary, we could remove those too